man/nvim.1

```diff
@@ -345,6 +351,11 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev VIM
+Used to locate various user files, such as the user-local nvimrc.
```

Comment:
Sure, I'll do that. I took the existing phrasing out of the tarsnap manuals, so maybe I was trying to hard to sound professional ;)